broker-cli/commands/order/cancel-all.js

```diff
@@ -0,0 +1,33 @@
+const BrokerDaemonClient = require('../../broker-daemon-client')
+const { handleError } = require('../../utils')
+
+/**
+ * sparkswap order cancel-all
+ *
+ * ex: `sparkswap order cancel-all --market btc/ltc'
+ *
+ * @param {Object} args
+ * @param {String} args.market
+ * @param {String} [rpcaddress] opts.rpcaddress
+ * @param {Logger} logger
+ */
+async function cancelAll (args, opts, logger) {
+  const {
+    rpcAddress,
+    market
+  } = opts
+
+  const request = {
+    market
+  }
+
+  try {
```

Comment:
@martineehrlich per our talk, we should add a prompt that asks the user if they _really_ want to cancel all their orders

Reply:
if we are going to have an interactive prompt, I think we also need a command line option to override it, e.g. `--force`. Otherwise you have to use it interactively, which is not great.